### ISSUE TITLE
Let custom items be dropped by mobs from the MythicMobs plug-in

### ISSUE THREAD

**Issue by knokko** [knokko/custom-items-gradle#40]
MythicMobs is a quite popular plug-in for adding custom mobs. These mobs can be configured to drop certain items, and it would be nice if these could be custom items.

**Comment by knokko** [comment]
Today, huge progress has been made for this. Currently, an example config like this is needed in MythicMobs:
```
DaedricSword:
  Id: NETHERITE_SWORD
  NBT:      
    KnokkosCustomItems: { Name: "daedric_sword" }
  Durability: 1
  Options:
    Unbreakable: true
```

(1) I should document this in the positive compatibility.

(2) The `Durability` and `Unbreakable` currently need to be configured explicitly, but users shouldn't have to. This is presumable due to a race condition with mythicmobs: I run the ItemUpdater right after a mob spawns, but the mythicmobs items are given after a 1 or 2 tick delay, which means the ItemUpdater won't see the items. I should do a similar run a few ticks after a mob spawns.

**Comment by knokko** [comment]
The custom items that are equipped by the bosses look fine, but they lose their texture when they are dropped by the mobs. Luckily, they will be 'revived' within 5 seconds by the item updater.

**Comment by knokko** [comment]
This will be fixed in Custom Items 9.2.

**Comment by knokko** [comment]
It looks like this feature doesn't work properly if Lib's Disguises (https://www.spigotmc.org/resources/libs-disguises-free.81/) or Interaction Visualizer (https://github.com/LOOHP/InteractionVisualizer). I should find out why or provide the Unbreakable + Damage work-around.

**Comment by knokko** [comment]
This will be solved in Custom Items 9.3.
